test/e2e/checkpoint_test.go

```diff
@@ -346,4 +348,48 @@ var _ = Describe("Podman checkpoint", func() {
 		Expect(podmanTest.NumberOfContainersRunning()).To(Equal(0))
 	})
 
+	// This test does the same steps which are necessary for migrating
+	// a container from one host to another
+	It("podman checkpoint container with export (migration)", func() {
+		// CRIU does not work with seccomp correctly on RHEL7
+		session := podmanTest.Podman([]string{"run", "-it", "--security-opt", "seccomp=unconfined", "-d", ALPINE, "top"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(0))
+
+		result := podmanTest.Podman([]string{"container", "checkpoint", "-l", "-e", "/tmp/checkpoint.tar.gz"})
+		result.WaitWithDefaultTimeout()
+
+		Expect(result.ExitCode()).To(Equal(0))
+		Expect(podmanTest.NumberOfContainersRunning()).To(Equal(0))
```

Comment:
Since I'm paranoid, I would also `Expect(podmanTest.NumberOfContainersRunning()).To(Equal(1))` after line 357.  I'm sure the checkpoint command would probably fail, but debugging would really suck if it didn't for some reason.

Reply:
I will add the `.To(Equal(1))` line.

Reply:
reminder ^^^